client_linux.go

```diff
@@ -544,3 +551,26 @@ func decodeSSID(b []byte) string {
 
 	return buf.String()
 }
+
+// decodeBSSLoad Decodes the BSSLoad IE. Supports Version 1 and Version 2
+// values according to https://raw.githubusercontent.com/wireshark/wireshark/master/epan/dissectors/packet-ieee80211.c
```

Comment:
Is there any documentation other than wireshark source code for these formats? Linking to a > 2MiB source file isn't my preference.

Reply:
Implementation in `iw scan` command is comparable in size. 

See [Line 1640ff in scan.c](https://git.kernel.org/pub/scm/linux/kernel/git/jberg/iw.git/tree/scan.c?h=main#n1640)  and [Line 1826](https://git.kernel.org/pub/scm/linux/kernel/git/jberg/iw.git/tree/scan.c?h=main#n1826) 

But I will do another search for better references.

Reply:
I also checked the `IEEE Std 802.11-2020` as a primary source. That document is approx 4000 Pages long, The Relevant definition of the `BSS Load` Information Element can be found in chapter 9.4.2.27. 

I will add the reference to the primary source and change the descriptions to closer match the language in the primary source to reduce any confusion. 

What do you think about that solution?

Reply:
LGTM